Remove pointless cast to fix build with base-gcc:

base/pcap-snoop.c:100: error: initializer element is not constant

Index: base/pcap-snoop.c
--- base/pcap-snoop.c.orig
+++ base/pcap-snoop.c
@@ -51,7 +51,9 @@
 
 #include <pcap.h>
 #include <unistd.h>
+#ifdef HAVE_PCAP_BPF_H
 #include <pcap-bpf.h>
+#endif
 #ifndef _WIN32
 #include <sys/param.h>
 #endif
@@ -95,7 +97,7 @@ int conn_freq = 100; // Number of packets after which 
                      // cleaning is performed
 int conn_ttl = 100;  // TTL of inactive connections in connection pool
 struct timeval last_packet_seen_time = // Timestamp of the last packet of the
-    (struct timeval) {0};              // last block of conn_freq packets seen
+    {0};                               // last block of conn_freq packets seen
 
 logger_mod *logger=NULL;
 
@@ -430,7 +432,7 @@ int main(argc,argv)
     
     if(!file){
       if(!interface_name){
-        if(pcap_findalldevs(&interfaces,errbuf)==-1) {
+        if(pcap_findalldevs(&interfaces,errbuf)==-1 || interfaces == NULL) {
           fprintf(stderr,"PCAP: %s\n",errbuf);
           err_exit("Aborting",-1);
         }
